dependencies64/ffmpeg/ffmpeg-build-linux/build-ubuntu.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+#compile ffmpeg successful on ubuntu 14.04.3 & 16.04.1 amd64 using the CasparCG build requirments described here:
+#https://raw.githubusercontent.com/casparcg/Server/2.1.0/BUILDING
 
-sudo apt-get install build-essential curl tar pkg-config
+sudo apt-get -y update
+sudo apt-get -y upgrade
+sudo apt-get -y install build-essential curl tar pkg-config python2.7-dev zlib1g-dev autoconf libtool subversion libfontconfig1-dev ant default-jre default-jdk frei0r-plugins-dev libv4l-dev
 
```

Comment:
Warning its not self sustaining! It still needs some dependencies casparcg server build uses for instance "libdrm" which is pulled by the casparcg server mesa/x dependencies. 

If you would like to build ffmpeg 100% on its own I can do another clean ubuntu install without the casparcg depencencies and will debug build dependencies for another X hours?